cypress/e2e/dashboard/events/events-index.cy.ts

```diff
@@ -297,23 +297,50 @@ describe('events dashboard', () => {
 
   it('chapter admin should be allowed to edit event, but nobody else', () => {
     const eventId = 1;
+
+    cy.logout();
```

Comment:
```suggestion
```
We only need to logout if we want to test how logged out users interact with the site.  Since we cy.login before asserting anything, this can go.